### DISCUSSION TITLE
Suggestion: Ship damage/health modeled on screens. (or holo projector)

### DISCUSSION THREAD

**Discussion by insanebotv21** [tsunamayo/Starship-EVO#4000] [Suggestions]
A small version of the ships blueprint with damaged parts highlighted; either slowly going through the layers or fixed to a layer. 
Even without voxel based damage this should still be possible to replicate since a ship already knows what needs to be repaired every time you repair it. it'd just be a matter of representation; it doesn't need to be a perfect representation to be extremely useful and immersive. 
An example of what i mean is found in this mod. 
https://steamcommunity.com/sharedfiles/filedetails/?id=715138160

**Comment by luckyrada** [comment]
this would be really cool
